source/2024.tex

```diff
@@ -0,0 +1,101 @@
+
+\subsection{UKPA Members}
+The UKPA membership is the electoral body to which the trustees are accountable.
+There are two classes of membership:
+
+\begin{enumerate}
+	\item Members: have the full rights and obligations described in the UKPA constitution, including the right to vote on members decisions.
+	\item Associate Members: non-voting class of membership as described in the UKPA constitution.
+\end{enumerate}
+
+\subsection{Trustee Appointment}
+At the AGM held on 24-Sep-2023, Vince Knight retired as a trustee in accordance with section 13.1.1 of the UKPA's constitution.
+
+Becky Smith also retired as a trustee in accordance with sections 13.1.4 and 13.1.5 of the constitution.
+
+Vincent Knight and Becky Smith were elected as trustees for a further term in accordance with section 13.1.4 of the constitution.
+
+On 04-Oct-2023, Vince Knight resigned as a trustee.
+
+By a resolution dated 10-Oct-2023, Daniele Procida was appointed as a trustee of the UKPA in accordance with clause 13.1.4 of the constitution
+\subsection{Risk and Internal Control}
+The trustees have identified the major risks to which they believe the UKPA is exposed. These are:
+
+\subsubsection{Reduction or Loss of Members}
+If there were a reduction in membership of the UKPA then there would have to be a contraction, consolidation or closure of its activities.
+
+
+\subsubsection{Reduction or Loss of Volunteers}
+The UKPA is totally reliant upon volunteers to run and administer the activities of the UKPA. If there were a reduction in the number of volunteers to an unacceptable level, then there would have to be a contraction, consolidation or closure of activities.
+
+\section{Achievements and Performance}
+PyCon UK 2023 was held at Cardiff City Hall from 22nd to 25th September 2023.
+
+\section{Financial Review}
+
+\subsection{Reserves Policy}
+The UKPA has undertaken a review of its potential liabilities in the face of foreseeable risks and determined that a reasonable sum to hold in reserve is that required to cover a year's operational expenditure plus some contingency for unforeseen costs: \pounds10K
+
+At the end of March 2024, the UKPA held reserves of \pounds TODO.
+
+All funds raised from previous PyCon UK conferences are currently held by PyCon UK Society Ltd (PUKSL). 
+
+Funds from the 2023 PyCon UK conference are held by the wholly owned subsidiary trading company, UKPA Trading Ltd (UKPATL). The intention is that funds held by PUKSL will be transferred to UKPATL and PUKSL will be closed.
+
+\subsection{Investment Policy}
+The UKPA has adopted a low risk strategy and all funds are held in cash using only mainstream banks or building societies.
+
+\section{Declaration}
+The trustees declare that they have approved the trustees report above.
+
+
+Signed on behalf of the charity's trustees:
+
+\begin{tabular}{l l}
+	\textbf{Signature:}\vspace{3cm}                       \\
+	\textbf{Date:}      & 14 September 2023\vspace{1cm} \\
```

Comment:
Date needs updated to 2024

Reply:
I've removed it for now until we know what the signature date will be